File: app/src/main/java/com/adiaz/deportesmadrid/utils/ListItem.java
package com.adiaz.deportesmadrid.utils;

import java.util.Comparator;

public class ListItem {

    String name;
    String count;

    public ListItem(String n, String i) {
        name = n;
        count = i;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCount() {
        return count;
    }

    public void setCount(String count) {
        this.count = count;
    }

    public static class ListItemCompartor implements Comparator<ListItem> {
        @Override
        public int compare(ListItem item01, ListItem item02) {
            if (item01==null || item02==null) {
                return 0;
            }
            return item01.getName().compareTo(item02.getName());
        }
    }
}

File: app/src/main/java/com/adiaz/deportesmadrid/utils/ListItem.kt
package com.adiaz.deportesmadrid.utils

import java.util.Comparator

class ListItem(var name: String, var count: String) {

    class ListItemCompartor : Comparator<ListItem> {
        override fun compare(item01: ListItem?, item02: ListItem?): Int {
            return if (item01 == null || item02 == null) {
                0
            } else item01.name.compareTo(item02.name)
        }
    }
}
